packages/chevrotain/src/parse/parser/parser.ts

```diff
@@ -65,7 +71,8 @@ export const DEFAULT_PARSER_CONFIG: IParserConfig = Object.freeze({
     dynamicTokensEnabled: false,
     outputCst: true,
     errorMessageProvider: defaultParserErrorProvider,
-    serializedGrammar: null
+    serializedGrammar: null,
+    nodePositionTracking: <"full" | "onlyStart" | "onlyOffset" | "none">"none"
```

Comment:
why do we need a cast here?

Reply:
if I don't cast, the error Type string is not assignable to type '"none" | "full ...' is thrown 

(sorry for the formatting, I'm on my phone...)